src/components/BottomBar/Origin/NodeNavigationButton.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import {
+  MdCenterFocusStrong,
+  MdFlashOn,
+  MdFlight
+} from 'react-icons/md';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import Focus from 'svg-react-loader?name=Focus!../../../icons/focus.svg';
+
+import { NavigationAimKey, NavigationAnchorKey, RetargetAnchorKey } from '../../../api/keys';
+import InfoBox from '../../common/InfoBox/InfoBox';
+import Button from '../../common/Input/Button/Button';
+import Row from '../../common/Row/Row';
+import SvgIcon from '../../common/SvgIcon/SvgIcon';
+
+import styles from './NodeNavigationButton.scss';
+
+const NavigationTypes = {
+  jump: 'jump',
+  fly: 'fly',
+  focus: 'focus',
+  frame: 'frame'
+};
+
+function NavigationButton({
+  identifier, onFinish, showLabel, type
+}) {
+  const luaApi = useSelector((state) => state.luaApi);
+
+  function focus(event) {
+    if (!event.shiftKey) {
+      luaApi.setPropertyValueSingle(RetargetAnchorKey, null);
+    }
+    luaApi.setPropertyValueSingle(NavigationAnchorKey, identifier);
+    luaApi.setPropertyValueSingle(NavigationAimKey, '');
+    event.stopPropagation();
+    onFinish();
+  }
+
+  const flyTo = (event) => {
+    if (event.shiftKey) {
+      luaApi.pathnavigation.flyTo(identifier, 0.0);
+    } else {
+      luaApi.pathnavigation.flyTo(identifier);
+    }
+    event.stopPropagation();
+    onFinish();
+  };
+
+  const zoomToFocus = (event) => {
+    if (event.shiftKey) {
+      luaApi.pathnavigation.createPath({
+        TargetType: 'Node',
+        Target: identifier,
+        Duration: 0,
+        PathType: 'Linear'
+      });
+    } else {
+      luaApi.pathnavigation.createPath({
+        TargetType: 'Node',
+        Target: identifier,
+        PathType: 'Linear'
+      });
+    }
+    event.stopPropagation();
+    onFinish();
+  };
+
+  const fadeTo = async (event) => {
+    event.stopPropagation();
+    onFinish();
+    luaApi.pathnavigation.jumpTo(identifier);
+  };
+
+  const buttonClass = showLabel ? styles.navButton : styles.smallNavButton;
+
+  const content = {
+    onClick: () => {},
+    icon: <> </>,
+    title: '',
+    info: undefined
+  };
+
+  switch (type) {
+    case NavigationTypes.jump:
+      content.onClick = fadeTo;
+      content.title = 'Jump to';
+      content.icon = <MdFlashOn className={styles.buttonIcon} />;
+      break;
+    case NavigationTypes.focus:
+      content.onClick = focus;
+      content.title = 'Focus';
+      content.icon = <SvgIcon className={styles.buttonIcon}><Focus /></SvgIcon>;
+      break;
+    case NavigationTypes.fly:
+      content.onClick = flyTo;
+      content.title = 'Fly to';
+      content.icon = <MdFlight className={styles.buttonIcon} />;
+      break;
+    case NavigationTypes.frame:
+      content.onClick = zoomToFocus;
+      content.title = 'Zoom to / Frame';
```

Comment:
I think it is sufficient to choose either of these titles. The slash makes it a bit confusing for me at least

Reply:
Any preference for either word @alexanderbock @ylvaselling ?  The reason there is two is because I couldn't decide 😅

Reply:
"Zoom to" on its own doesn't really make sense to me since if we're close to an object it leads to a "zoom out" motion

Reply:
Maybe "Overview" ?

Reply:
Conclusion:  Keep the name for now, and long term fix the camera path system so that we don't need two buttons at all for this